api/controllers/game/editmode.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (req, res) {
+  const { isRanked } = req.body;
+  const promiseGame = gameService.findGame({ gameId: req.session.game });
+  const promisePlayer = userService.findUser({ userId: req.session.usr });
+  Promise.all([promiseGame, promisePlayer])
+    .then(function changeAndSave(values) {
+      const [game, player] = values;
```

Comment:
Generally prefer `async/await` going forward. Additionally, the data that's used from the `player` here also exists on the session, which is quicker to query. So we can just query for the game here
```suggestion
module.exports = function (req, res) {
  const { isRanked } = req.body;
  const game = await gameService.findGame({ gameId: req.session.game });

```